service/update_configuration_test.go

```diff
@@ -0,0 +1,95 @@
+package service
+
+import (
+	"bytes"
+	"github.com/gorilla/mux"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+func TestService_UpdateConfiguration(t *testing.T) {
```

Comment:
Rename this function with proper format. Something like TestServiceUpdateConfiguration

Reply:
Using TestService_<function_name> as the convention in unit tests.